src/languageFeatures.ts

```diff
@@ -626,3 +626,48 @@ export class FormatOnTypeAdapter extends FormatHelper implements monaco.language
 		});
 	}
 }
+
+// --- rename ----
+
+export class RenameAdapter extends Adapter implements monaco.languages.RenameProvider {

```

Comment:
There is another optional method `resolveRenameLocation` in the interface which I didn't implement. What is the purpose of this method and do we need that?